templates/patroni.yml.j2

```diff
@@ -0,0 +1,46 @@
+#########################################################################################
+# [ WARNING ]
+# patroni configuration file maintained by the postgres-operator
+# local changes may be overwritten.
+#########################################################################################
+scope: {{ scope }}
+namespace: /db/
+name: {{ member_name }}
+
+restapi:
+  listen: '{{ self_ip }}:8008'
+  connect_address: '{{ self_ip }}:8008'
+
+raft:
+  data_dir: {{ conf_path }}/raft
+  self_addr: '{{ self_ip }}:2222'
+
+bootstrap:
+  dcs:
+    ttl: 30
+    loop_wait: 10
+    retry_timeout: 10
+    maximum_lag_on_failover: 1048576
+    postgresql:
+      use_pg_rewind: true
+
+  initdb:
+  - encoding: UTF8
+  - data-checksums
+
+  pg_hba:
+  - host replication replication 127.0.0.1/32 md5
+  - host all all 0.0.0.0/0 md5
+postgresql:
+  listen: '{{ self_ip }}:5432'
+  connect_address: '{{ self_ip }}:5432'
+  bin_dir: /usr/lib/postgresql/{{ version }}/bin/
+  data_dir: {{ conf_path }}/pgdata
+  pgpass: /tmp/pgpass
+  authentication:
+    replication:
+      username: replication
+      password: {{ replication_password }}
+    superuser:
+      username: postgres
+      password: {{ superuser_password }}
```

Comment:
It may be useful to comment some of the non-obvious lines here, it will serve as useful documentation for people trying to understand how the charm behaves. Another option is to just drop a link to the Patroni docs in a comment at the op if they're any good ;-)